exchanges/binance/binance_wrapper.go

```diff
@@ -492,9 +490,10 @@ func (b *Binance) UpdateTickers(ctx context.Context, a asset.Item) error {
 		}
 
 		for y := range tick {
-			cp, err := currency.NewPairFromString(tick[y].Symbol)
+			tick[y].Symbol = strings.Replace(tick[y].Symbol, currency.UnderscoreDelimiter, "", 1)
+			cp, err := enabledPairs.DeriveFrom(tick[y].Symbol, "")
```

Comment:
I think there was a reason I did this and also for Bitmex I will add some comments, some pairs come back with delimiters and some don't and if I shifted that filtering to the function and it didn't have a delimiter it would error and not be added to the list.